src/commands/paas/elasticsearch/dump.ts

```diff
@@ -0,0 +1,185 @@
+import path from "path";
+import fs from "node:fs/promises";
+
+import { flags } from "@oclif/command";
+
+import { PaasKommand } from "../../../support/PaasKommand";
+
+/**
+  * Results of the document dump action.
+  */
+type DocumentDump = {
+  pit_id: string;
+  hits: DocumentDumpHits;
+};
+
+type DocumentDumpHits = {
+  total: DocumentDumpHitsTotal;
+  hits: DocumentDumpHit[];
+};
+
+type DocumentDumpHitsTotal = {
+  value: number;
+};
+
+type DocumentDumpHit = {
+  sort: string[];
+};
+
+class PaasInit extends PaasKommand {
+  public static description = "Dump data from the Elasticsearch of a PaaS application";
+
+  public static flags = {
+    help: flags.help(),
+    project: flags.string({
+      description: "Current PaaS project",
+    }),
+  };
+
+  static args = [
+    {
+      name: "environment",
+      description: "Project environment name",
+      required: true,
+    },
+    {
+      name: "applicationId",
+      description: "Application Identifier",
+      required: true,
+    },
+    {
+      name: "dumpDirectory",
+      description: "Directory where to store dump files",
+      required: true,
+    }
+  ];
+
+  async runSafe() {
+    // Log in to the PaaS
+    const apiKey = await this.getCredentials();
+
+    await this.initPaasClient({ apiKey });
+
+    const user = await this.paas.auth.getCurrentUser();
+    this.logInfo(
+      `Logged as "${user._id}" for project "${this.flags.project || this.getProject()
+      }"`
+    );
+
+    // Create the dump directory
+    await fs.mkdir(this.args.dumpDirectory, { recursive: true });
```

Comment:
does option { recursive true } works as mkdirp ?

Reply:
Yes, it works the same as `mkdir -p`